src/libs/ReportActionsUtils.js

```diff
@@ -547,6 +547,20 @@ function getIOUReportIDFromReportActionPreview(reportAction) {
     return lodashGet(reportAction, 'originalMessage.linkedReportID', '');
 }
 
+/**
+ * Find the reportAction whose childReportID is taskReportID in reportActions with reportID
+ *
+ * @param {String} taskReportID
+ * @param {String} reportID
+ * @returns {Object}
+ */
+function getParentReportActionForTask(taskReportID, reportID) {
+    return _.find(
+        allReportActions[reportID],
+        (reportAction) => reportAction && `${reportAction.childReportID}` === `${taskReportID}` && reportAction.actionName === CONST.REPORT.ACTIONS.TYPE.ADDCOMMENT,
```

Comment:
Why we are converting those into strings and not just `reportAction.childReportID === taskReportID`?

Reply:
Since `childReportID` is a number, while `reportID` is a string. I think we need to unify these but this would require a lot of work and beyond the scope of this issue.

Reply:
None is supposed to a be number. All report ids are supposed to be strings. please remove this extra string construction